### PR TITLE
File structure and CLI with tests

### PR DESCRIPTION
**Pull request by ahemphill**
Should I update .github/workflows/python_tools.yml?

### CODE REVIEW

File: tools/scoring/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Install
+To install the dependencies, please run the `python3 setup.py install` from the following directories, in order:
+* digitalbuildings/tools/validators/ontology_validator
+* digitalbuildings/tools/validators/instance_validator
+* digitalbuildings/tools/scoring
+
+## Usage
+
+From the `scoring` directory: `python3 score/cli.py -ont path/to/ontology/yaml/resources -prop path/to/proposed/file.yaml -sol path/to/solution/file.yaml -v True`
```
Comment (charbull) [inline tools/scoring/README.md:13]: can we please rename this `cli.py` to `score.py` just to be compliant with the other tools on github

File: tools/scoring/setup.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+""" Setup file for the configuration scoring tool."""
+
+from setuptools import setup, find_packages
+
+setup(
+    name='scoring',
+    version='0.0.1',
+    url='https://github.com/google/digitalbuildings',
+    license='Apache License',
+    author='',
```
Comment (charbull) [inline tools/scoring/setup.py:24]: add your name ! it is your work :)

**Review by charbull — APPROVED** [review]
approve: just minor rename :)

**Review by charbull — COMMENTED** [review]
can you please have the following directory

tools/scorer/
    scorer.py // this is the cli
    score/ // here all the code is placed
    tests/ //here all the test files

File: tools/scoring/README.md (hunk 1 above)
Comment (ahemphill) [reply tools/scoring/README.md:13]: I tried to implement the naming scheme you suggested but kept encountering issues with module resolution (due to naming conflicts, I believe). I will revisit.

**Pull request merged**